examples/commander.yaml

```diff
@@ -20,3 +20,7 @@ tests:
     command: invalid
     stderr: "/bin/sh: 1: command invalid: not found"
     exit-code: 127
+
+  it should skip:
+    command: echo hello
+    disable: true
```

Comment:
Would prefer to use `skip: true` to be consistent with the source code. :+1:

Reply:
I have a commit staged on my local, using `enabled` instead of `disable`. Would you like to see that? I was trying to be consistent with #127. The idea mainly stemmed from some helm repos `values.yaml`'s I've configured

Reply:
Actually, thinking about the `enable` implementation. It is a little more complex, because `YAML.Test` has to be 
```golang
type YAMLTest struct {
	Title    string             `yaml:"-"`
	Command  string             `yaml:"command,omitempty"`
	ExitCode int                `yaml:"exit-code"`
	Stdout   interface{}        `yaml:"stdout,omitempty"`
	Stderr   interface{}        `yaml:"stderr,omitempty"`
	Config   YAMLTestConfigConf `yaml:"config,omitempty"`
	Enable  *bool               `yaml:"enable,omitempty"`
}
```
It the requires a little bit more logic to convert this into the proper Boolean for `runtime.TestCase`.

In short, I'll move it to skip. That way the default is the zero value of Boolean.

Reply:
Actually, thinking about the `enable` implementation. It is a little more complex, because `YAML.Test` has to be 
```golang
type YAMLTest struct {
	Title    string             `yaml:"-"`
	Command  string             `yaml:"command,omitempty"`
	ExitCode int                `yaml:"exit-code"`
	Stdout   interface{}        `yaml:"stdout,omitempty"`
	Stderr   interface{}        `yaml:"stderr,omitempty"`
	Config   YAMLTestConfigConf `yaml:"config,omitempty"`
	Enable  *bool               `yaml:"enable,omitempty"`
}
```
It the requires a little bit more logic to convert this into the proper Boolean for `runtime.TestCase`.

In short, I'll move it to skip. That way the default is the zero value of Boolean.

Reply:
Imho it is totally fine to name the propety `disabled`, `skip` or `ignore`, just keep it in line with the source. 
I don't like `enabled` that much because it is `true` by default and need always inversed  checks like `!test.Enabled` or `test.Enabled == false`.
Based on your code I would prefer `skip` as a property in the YAML spec.
btw you're implementation looks good to me :+1:

Reply:
Got it,

Yes, I always struggle with these highly conditional features, however your `EventHandler` made it feel very nice.